### PR TITLE
Add debugging print to check aarch64 repl test

### PR DESCRIPTION
**Pull request by gbaraldi**

### CODE REVIEW

**Comment by vtjnash** [comment]
```
      From worker 10:	tmpdir = "/cache/build/default-armageddon-7/julialang/julia-master/tmp/jl_18tn9T"
      From worker 10:	 pwd() = "/cache/build/default-armageddon-7/julialang/julia-master/tmp/jl_18tn9T"
```

**Comment by vtjnash** [comment]
```
      From worker 4:	stat(tmpdir) = StatStruct("." size: 4096 bytes device: 66307 inode: 54934720 mode: 0o040700 (drwx------) nlink: 2 uid: 1000 (juliaci) gid: 1000 (juliaci) rdev: 0 blksz: 4096 blocks: 8 mtime: 2022-09-28T08:11:37+0000 (just now) ctime: 2022-09-28T08:11:37+0000 (just now))
      From worker 4:	   stat(".") = StatStruct("." size: 4096 bytes device: 66307 inode: 54934720 mode: 0o040700 (drwx------) nlink: 2 uid: 1000 (juliaci) gid: 1000 (juliaci) rdev: 0 blksz: 4096 blocks: 8 mtime: 2022-09-28T08:11:37+0000 (just now) ctime: 2022-09-28T08:11:37+0000 (just now))
      From worker 4: 	 stat(pwd()) = StatStruct("." size: 4096 bytes device: 66307 inode: 54934720 mode: 0o040700 (drwx------) nlink: 2 uid: 1000 (juliaci) gid: 1000 (juliaci) rdev: 0 blksz: 4096 blocks: 8 mtime: 2022-09-28T08:11:37+0000 (just now) ctime: 2022-09-28T08:11:37+0000 (just now))
```

**Comment by vtjnash** [comment]
The first call to `stat` returns info on the wrong folder:
`
      From worker 4:	infoa = StatStruct("." size: 4096 bytes device: 66307 inode: 3700025 mode: 0o040775 (drwxrwxr-x) nlink: 15 uid: 1000 (juliaci) gid: 1000 (juliaci) rdev: 0 blksz: 4096 blocks: 8 mtime: 2022-09-28T16:36:41+0000 (1 minute ago) ctime: 2022-09-28T16:38:03+0000 (14 seconds ago))
`

note how old this folder was when last modified (1 minute), last accessed (14 s), and how many hard links it has (probably folders inside)

**Comment by gbaraldi** [comment]
So for some reason `.` doesn't update fast enough for the test? Because it does give the correct result after it.

**Comment by gbaraldi** [comment]
What were you trying to test in the last commit @vtjnash ?

**Comment by vtjnash** [comment]
I am trying to isolate which lines are important and which don't change the result

**Comment by gbaraldi** [comment]
`From worker 4:	realpath(".") = "/cache/build/default-armageddon-5/julialang/julia-master/julia-3e628d6290/share/julia/test"`
`From worker 4:	realpath(tmpdir) = "/cache/build/default-armageddon-5/julialang/julia-master/tmp/jl_Rmtz2C"`

**Comment by gbaraldi** [comment]
Yeah, the expansion of `.` didn't change quickly enough for this test. Which is odd.

File: stdlib/REPL/test/repl.jl [hunk 1]
```diff
@@ -150,1440 +98,21 @@ fake_repl(options = REPL.Options(confirm_exit=false,hascolor=true)) do stdin_wri
             readuntil(stdout_read, tmpdir_pwd)
             readuntil(stdout_read, "\n")
             readuntil(stdout_read, "\n")
```
Comment (vtjnash) [inline stdlib/REPL/test/repl.jl:100]: A little part of me wonders if the path on aarch64 is just a little bit too long, and causes tmpdir_pwd to be printed several times here. e.g. the same as #44972. The order of these reads is a bit nonsensical. Thus the actual `cd` might not have happened yet, until the added `@show` cause a context switch.

(you can prove this theory by checking when `get(ENV, "OLDPWD", "")` gets set here)

Comment (gbaraldi) [reply stdlib/REPL/test/repl.jl:100]: Yeah, it's as you expected. A context switch is what triggers it to change. Not sure what the fix for this is. The paths don't look too too long

**Comment by vtjnash** [comment]
Fixed by https://github.com/JuliaLang/julia/pull/47160#pullrequestreview-1142282453

**Pull request closed without merging**